assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -1222,6 +1223,14 @@ export default Component.extend({
     }
   },
 
+  @action
+  onHoverMessage(message) {
+    this.set(
+      "hoveredMessageId",
+      message?.id && message.id !== this.hoveredMessageId ? message.id : null
```

Comment:
This is difficult to read but.. if it works that's ok

Reply:
yep there are various ways to write it, I finally stopped on this one, I think that's okayish.